### PR TITLE
fix(split-button): don't render the menu if there are no menu items

### PR DESCRIPTION
**Pull request by Kiarokh**
fix https://github.com/Lundalogik/crm-feature/issues/3086 

## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

**Review by Befkadu1 — DISMISSED** [review]

**Review by Befkadu1 — APPROVED** [review]

File: src/components/split-button/split-button.tsx [hunk 1]
```diff
@@ -61,30 +61,44 @@ export class SplitButton {
 
     render() {
         return (
-            <Host>
+            <Host
+                class={{
+                    'has-menu': !!this.items.length,
```
Comment (Befkadu1) [inline src/components/split-button/split-button.tsx:66]: Can we check this one?

**Review by Befkadu1 — CHANGES_REQUESTED** [review]
Can we check the comment I gave you above

File: src/components/split-button/split-button.tsx (hunk 1 above)
Comment (Kiarokh) [reply src/components/split-button/split-button.tsx:66]: what do you mean?

**Review by Befkadu1 — APPROVED** [review]

File: src/components/split-button/split-button.tsx (hunk 1 above)
Comment (Befkadu1) [reply src/components/split-button/split-button.tsx:66]: Sorry you were right!

**Pull request merged**